payloads/extensions/linux_mount.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# LINUX_MOUNT v1 by @drapl0n
+# Auto mounts BashBunny on GNU/Linux systems.
+# NOTE: Mount path is stored in variable "lmnt".
+# Usage: LINUX_MOUNT - to automatically mount BashBunny.
+#	 LINUX_UMOUNT - to unmount mounted BashBunny.
+
+function LINUX_MOUNT() {
+	Q DELAY 1000
+	Q CTRL-ALT t
+	Q DELAY 1000
+	Q STRING unset HISTFILE
+	Q ENTER
+	Q DELAY 1000
```

Comment:
Why is this DELAY so long?

Reply:
depending on various systems with different hardware configuration, it takes average time of a second to load terminal emulator.

Reply:
I suggest removing the starting DELAY here and allow the user the call their own DELAY in their payload before calling the function.

Reply:
This comment was asking why wait a full second after running "unset HISTFILE"

Reply:
yes I will remove that, I added that delay during testing phase.